app/models/container_image.rb

```diff
@@ -24,6 +24,7 @@ class ContainerImage < ApplicationRecord
   has_many :openscap_rule_results, :through => :openscap_result
   has_many :labels, -> { where(:section => "labels") }, :class_name => CustomAttribute, :as => :resource, :dependent => :destroy
   has_many :docker_labels, -> { where(:section => "docker_labels") }, :class_name => CustomAttribute, :as => :resource, :dependent => :destroy
+  has_many :openshift_annotations, -> { where(:section => "openshift_annotations") }, :class_name => CustomAttribute, :as => :resource, :dependent => :destroy
```

Comment:
I chose this name to clarify that the annotations are coming from Openshift and not from ManageIQ or Docker. I know that we currently only have annotations from Openshift but I felt that it is still better to have it clearly.